lib/fastlane/plugin/firebase_app_distribution/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module FirebaseAppDistribution
-    VERSION = "0.5.0"
+    VERSION = "0.5.1"
```

Comment:
Not a bug fix, should it be a minor bump?

Reply:
You're right. Done.